### PR TITLE
Input text key component

### PR DESCRIPTION
**Pull request by youssef-night**

### CODE REVIEW

File: libs/ui/components/contents/icon/icons/copy-clipboard.tsx [hunk 1]
```diff
@@ -23,4 +23,22 @@ const CopyClipBoard = (props: React.SVGProps<SVGSVGElement>) => (
         </defs>
     </svg>
 );
-export default CopyClipBoard;
+
+export const CopyClipBoardLarge = (props: React.SVGProps<SVGSVGElement>) => (
```
Comment (ThomasNight) [inline libs/ui/components/contents/icon/icons/copy-clipboard.tsx:27]: one file for each icon

File: libs/ui/components/contents/icon/icons/copy-clipboard.tsx [hunk 2]
```diff
@@ -23,4 +23,22 @@ const CopyClipBoard = (props: React.SVGProps<SVGSVGElement>) => (
...
+    <svg
+        {...props}
+        width={props?.width ?? '28'}
+        height={props?.height ?? '28'}
+        viewBox={props?.viewBox ?? '0 0 28 28'}
```
Comment (ThomasNight) [inline libs/ui/components/contents/icon/icons/copy-clipboard.tsx:32]: we never change the view box

File: libs/ui/components/contents/icon/icons/copy-clipboard.tsx [hunk 3]
```diff
@@ -23,4 +23,22 @@ const CopyClipBoard = (props: React.SVGProps<SVGSVGElement>) => (
...
+                fill={'#8F95B2'}
```
Comment (ThomasNight) [inline libs/ui/components/contents/icon/icons/copy-clipboard.tsx:40]:
```suggestion
                fill={props.fill ?? '#8F95B2'}
```

File: libs/ui/components/contents/icon/icons/index.tsx [hunk 4]
```diff
@@ -30,12 +31,15 @@ const Icons = {
     send: Send,
     more: More,
     copy: CopyClipBoard,
+    copyLarge: CopyClipBoardLarge,
```
Comment (ThomasNight) [inline libs/ui/components/contents/icon/icons/index.tsx:34]: keep same names

File: libs/ui/components/contents/icon/icons/projectLine.tsx [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    <svg
+        width={props?.width ?? '22'}
+        height={props?.height ?? '22'}
+        viewBox={props?.viewBox ?? '0 0 22 22'}
+        fill='none'
+        xmlns='http://www.w3.org/2000/svg'
+    >
+        <path
+            d='M1 1V9C1 15.6274 6.37258 21 13 21H21'
+            stroke='#E7E9F4'
```
Comment (ThomasNight) [inline libs/ui/components/contents/icon/icons/projectLine.tsx:13]:
```suggestion
            stroke={props.stroke ?? '#E7E9F4'}
```

**Comment by youssef-night** [comment]
https://app.clickup.com/t/861mdmqc2

File: libs/ui/components/inputs/button-icon/index.stories.tsx [hunk 6]
```diff
@@ -19,7 +19,9 @@ const props: IButtonIconProps = {
     hoverBackgroundColor: '#4C36B5',
     border: 'none',
     borderRadius: '8px',
-    icon: <Icon name='add' stroke='white' width='16' height='16' />,
+    displayIcon: (isHovered) => (
```
Comment (ThomasNight) [inline libs/ui/components/inputs/button-icon/index.stories.tsx:22]:
```suggestion
    displayIcon: () => (
```

don't take unused params

Comment (ThomasNight) [reply libs/ui/components/inputs/button-icon/index.stories.tsx:22]: Remove it everywhere

File: libs/ui/components/inputs/button-icon/index.tsx [hunk 7]
```diff
@@ -18,7 +21,9 @@ export default function ButtonIcon({
             minHeight={'28px'}
             minWidth={'28px'}
             onClick={handleOnClick}
-            icon={icon}
+            onMouseOver={updateHoverState}
```
Comment (ThomasNight) [inline libs/ui/components/inputs/button-icon/index.tsx:24]: setIsHover = true

File: libs/ui/components/inputs/button-icon/index.tsx [hunk 8]
```diff
@@ -18,7 +21,9 @@ export default function ButtonIcon({
...
+            onMouseLeave={updateHoverState}
```
Comment (ThomasNight) [inline libs/ui/components/inputs/button-icon/index.tsx:25]: setIsHover = false

File: libs/ui/components/inputs/input-project-key/index.tsx [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+                        padding='12px 8px 12px 12px'
+                        borderRight={`1px solid ${COLORS.Tag.value}`}
+                        borderTopLeftRadius={borderRadius}
+                        borderBottomLeftRadius={borderRadius}
+                    >
+                        <Text
+                            {...FONTS.T1.T12px.SemiBold600}
+                            color={COLORS.InputText.value}
+                        >
+                            Projet key
```
Comment (ThomasNight) [inline libs/ui/components/inputs/input-project-key/index.tsx:49]: trad

File: libs/ui/components/inputs/input-project-key/index.tsx [hunk 10]
```diff
@@ -0,0 +1,88 @@
...
+    placeholder,
+    value,
+    rightElement,
+}: InputProjectKeyProps) {
+    // Render
+    return (
+        <>
+            <HStack>
+                {/* ICON SECTION */}
+                <Box mb='20px' ml='20px'>
```
Comment (ThomasNight) [inline libs/ui/components/inputs/input-project-key/index.tsx:32]: rem everywhere

**Review by ThomasNight — CHANGES_REQUESTED** [review]

File: libs/ui/components/contents/icon/icons/index.tsx (hunk 4 above)
Comment (ThomasNight) [reply libs/ui/components/contents/icon/icons/index.tsx:34]: todo: 
```ts
copyClipBoard: CopyClipBoard,
copyClipBoardLarge: CopyClipBoardLarge,
```

File: libs/ui/components/inputs/button-icon/index.stories.tsx (hunk 6 above)
Comment (ThomasNight) [reply libs/ui/components/inputs/button-icon/index.stories.tsx:22]: todo

File: libs/ui/components/inputs/input-project-key/index.stories.tsx [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+) => <InputProjectKey {...props} />;
+
+export const ProjectKeyInput = Template.bind({});
+ProjectKeyInput.args = {
+    color: COLORS.Text.T500.value,
+    border: `0.0625rem solid ${COLORS.Stroke.value}`,
+    borderRadius: '0.5rem',
+    padding: '0rem',
+    w: '36.25rem',
+    h: '40x',
```
Comment (ThomasNight) [inline libs/ui/components/inputs/input-project-key/index.stories.tsx:26]: fix height

**Review by ThomasNight — APPROVED** [review]

**Comment by ThomasNight** [comment]
fix conflicts

**Review by ThomasNight — APPROVED** [review]

**Pull request merged**